repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/servlet/http/HttpServletResponseWrapper.scala

```diff
@@ -161,40 +184,74 @@ class HttpServletResponseWrapper(originalResponse: HttpServletResponse, headerMo
 
     // If we are not in a mutable mode, immediately send error to the wrapped response.
     if (headerMode != ResponseMode.MUTABLE && bodyMode != ResponseMode.MUTABLE) {
-      originalResponse.sendError(i, s)
+      sendErrorFunc
     }
   }
 
-  def isError: Boolean = sentError
+  def isError: Boolean = {
```

Comment:
I don't know that there is a lot of value in adding logging into what is effectively getters and setters.